install.bat

```diff
@@ -0,0 +1,168 @@
+:: Name:     install.bat
+:: Purpose:  Builds Windows Installer for Firefox with uProxy
+:: Version: 0.1.1
+:: uProxy version: 0.8.39
+
```

Comment:
This is one of the cleanest batch files I've ever seen.  I'm genuinely impressed.